### ISSUE TITLE
Error: field label not supported: name

### ISSUE THREAD

**Issue by zengxiaobo123** [open-cluster-management-io/ocm#136]
**Describe the bug**
An error occurred when specifying to view ManifestWork

**To Reproduce**
An error occurred when specifying to view ManifestWork
```
telecom@npmaster1:~/zxb$ clusteradm get works --cluster nephio2
<ManifestWork>
└── <my-first-work>
│   ├── <Applied> True
│   ├── <Available> True
│   ├── <Cluster> nephio2
│   ├── <Number of Manifests> 2
└── <my-second-work>
    └── <Cluster> nephio2
    └── <Number of Manifests> 2
    └── <Applied> True
    └── <Available> True

telecom@npmaster1:~/zxb$ clusteradm get works my-first-work --cluster nephio2
Error: field label not supported: name
```
**Expected behavior**
I followed the steps of the official website to check the situation of the designated My-first-work I deployed, but this error occurred as a result. Is this my wrong operation in any step? If I made a mistake, please help me point it out.
This is my deployment file, and I just changed the cluster namespace：
```
apiVersion: work.open-cluster-management.io/v1
kind: ManifestWork
metadata:
  namespace: nephio2
  name: my-second-work
spec:
  workload:
    manifests:
      - apiVersion: v1
        kind: ServiceAccount
        metadata:
          namespace: default
          name: my-second-sa
      - apiVersion: apps/v1
        kind: Deployment
        metadata:
          namespace: default
          name: nginx-deployment
          labels:
            app: nginx
        spec:
          replicas: 3
          selector:
            matchLabels:
              app: nginx
          template:
            metadata:
              labels:
                app: nginx
            spec:
              serviceAccountName: my-second-sa
              containers:
                - name: nginx
                  image: nginx:1.14.2
                  ports:
                    - containerPort: 80
```
**Environment ie: OCM version, Kubernetes version and provider:**
```
telecom@npmaster1:~/zxb$ kubectl get node -A
NAME         STATUS   ROLES                  AGE    VERSION
master       Ready    control-plane,master   170d   v1.22.9
npworker11   Ready    <none>                 170d   v1.22.9
npworker12   Ready    <none>                 170d   v1.22.9

telecom@npmaster1:~/zxb$ clusteradm version
client          version :v0.5.1

server release  version :v1.22.9
default bundle  version :0.10.0

**Comment by mikeshng** [comment]
https://github.com/open-cluster-management-io/clusteradm/blob/v0.5.1/pkg/cmd/get/work/exec.go#L73-L74

Not sure why we are doing a List here instead of a Get
@ycyaoxdu any ideas? Thanks.

**Comment by qiujian16** [comment]
which version are you using? This should have been fixed in the latest code

**Comment by ycyaoxdu** [comment]
@zengxiaobo123 hi, you may confirm whether we could close this issue.

**Comment by qiujian16** [comment]
this is fixed in v0.11.0
/close
